tests/unit/test_util.cpp

```diff
@@ -78,3 +78,37 @@ TEST_F(UtilTest, testFromHex)
 
     ASSERT_THROW(result = mococrw::utility::fromHex("  deadbeef"), mococrw::MoCOCrWException);
 }
+
+TEST_F(UtilTest, stringCleanse)
+{
+    std::string secret{"1234"};
+    mococrw::utility::stringCleanse(secret);
+    ASSERT_THAT(secret, Eq(std::string()));
+}
+
+TEST_F(UtilTest, stringCleanseCheckMemory)
+{
+    std::string secret{"1234"};
+    auto secretSize = secret.size();
+    std::string zeroedOutString(secretSize, '\0');
+    mococrw::utility::stringCleanse(secret);
+    ASSERT_GE(secret.capacity(), secretSize);
```

Comment:
This tests something that isn't actually guaranteed.
The reference says

> Unlike for [std::vector::clear](https://en.cppreference.com/w/cpp/container/vector/clear), the C++ standard does not explicitly require that [capacity](https://en.cppreference.com/w/cpp/string/basic_string/capacity) is unchanged by this function, but existing implementations do not change capacity. This means that they do not release the allocated memory (see also [shrink_to_fit](https://en.cppreference.com/w/cpp/string/basic_string/shrink_to_fit)).

Reply:
Done